tags/Generic/TaxonomiesTracker/README.md

```diff
@@ -0,0 +1,17 @@
+# Content Taxonomies Tracker
+
+Fires a custom `taxonomyTracker` event for every taxonomy value in the given array.
+
```

Comment:
You might want to specify that we're talking about `dataLayer.push()`, so something like

Takes an array of values as an input, and fires a `dataLayer.push()` for each value in the array. This way you can process every item in an array (e.g. taxonomy values) as its own `dataLayer` event, firing a tag for each, for example.

Reply:
Good suggestion. I've done so